src/engine/QMLEngine.js

```diff
@@ -141,6 +141,10 @@ class QMLEngine {
     return basePath.join("/");
   }
 
+  extractFileName(file) {
+    return file.split('\\').pop().split('/').pop();
```

Comment:
@hhaveri Well, I meant a single `split` istead of two ones. `pop()` is still needed here =).